polkadot-parachains/integritee-runtime/Cargo.toml

```diff
@@ -118,9 +120,6 @@ std = [
 	"xcm-executor/std",
 	"pallet-aura/std",
 	"sp-consensus-aura/std",
-	# benchmarking stuff
-	"frame-benchmarking/std",
```

Comment:
These imports broke the build and the encointer-parachain does not import them either in the `std` feature, so I removed them.

Reply:
I don't think this is legit. Encointer para doesn't have this because it is **not yet** benchmarked. @clangenb I think you had to change the way benchmarks are included in the runtime, right?

Reply:
Hmm, I don't know if this was different at some point, or if this is a mistake that has been introduced in this node. However, removing this now seems to be correct.